src/Microsoft.Tye.Core/ConfigFileFinder.cs

```diff
@@ -23,12 +23,14 @@ public static bool TryFindSupportedFile(string directoryPath, out string? filePa
                     return true;
                 }
 
-                if (files.Length > 1)
+                if (files.Length <= 1)
```

Comment:
Isn't this condition only applicable when the length is 0? If so let's just make it `if (files.Length == 0)`